src/core/annotation.js

```diff
@@ -1516,6 +1623,23 @@ class TextWidgetAnnotation extends WidgetAnnotation {
 
     return chunks;
   }
+
+  getFieldObject() {
+    return {
+      id: this.data.id,
+      value: this.data.fieldValue,
+      multiline: this.data.multiline,
```

Comment:
For consistency, shouldn't the key have the same capitalization as well (i.e. `multiLine`)?
Having inconsistent casing does seem like a good way of *accidentally* introducing bugs elsewhere.

Reply:
On js side multiline property is lowercase:
https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/js_api_reference.pdf#page=433
So here the keys reflect what we've in the js api.
I'd prefer changing the key names in `this.data` for consistency, but it'd broke the api (pdf.js' one)...
@Snuffleupagus, I probably made this typo because of this mismatch so I strongly agree with you.

Reply:
It's indeed not possible to change this now because it's an API-breaking change.